examples/minimal-sveltekit/src/routes/+page.svelte

```diff
@@ -0,0 +1,50 @@
+<script lang="ts">
+  import { createUploader, useUploadThing } from "$lib/utils/uploadthing";
+
+  import { Uploader } from "@uploadthing/svelte";
+
+  import "@fontsource-variable/inter";
+
+  const uploader = createUploader({
+    endpoint: "videoAndImage",
```

Comment:
Nit - but to keep consistency in the APIs, shouldn't endpoint be a positional argument here?